esp/public/media/scripts/datepicker-duration.js

```diff
@@ -0,0 +1,42 @@
+function setupDatepickerDurationLabel(widget, id, mediaUrl, dateFormat, timeFormat){
+  var $picker = $j("#" + id);
+  var $info = $j("#" + id + "-info");
+  var $gi = $j("#" + id + "-gi");
+  var $duration = $j("#" + id + "-duration");
+  function refresh() {
+    var date = $picker[widget]('getDate');
+    if (date) {
+      var hours = (date - Date.now()) / (60*60*1000);
+      var abshours = Math.abs(hours);
+      var duration = (abshours < 24) ? (
+        abshours.toFixed(1) + " hours"
+      ) : (
+        (abshours / 24).toFixed(1) + " days"
+      );
+      if (abshours > 24 * 120) {
+        $info.prop('class', 'label label-important');
+        $gi.prop('class', 'glyphicon glyphicon-alert');
+      } else {
+        if (hours < 0) {
+          $info.prop('class', 'label label-success');
```

Comment:
What's the rationale behind these colors, or is it arbitrary? (It's also a bit confusing, because I want to think "green" means "deadline open" but this isn't consistent because right now green in the open date means deadline open and green in the closed date means deadline closed. Not sure you can do anything about that though

Reply:
Fair. I got rid of the green/blue colors; now it's just blue if close and red if far away.